content/en/examples/application/guestbook/mongo-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1 #  for k8s versions before 1.9.0 use apps/v1beta2  and before 1.8.0 use extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mongo
+  labels:
+    app: mongo
```

Comment:
Ideally, use the [recommended app label keys](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)